### PR TITLE
Update website for Minetest 5.2.0

### PR DESCRIPTION
**Pull request by rubenwardy**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -24,7 +24,7 @@ <h1 class="title">Minetest</h1>
           <a class="button is-primary is-size-5" href="/downloads/">Download</a>
         </div>
         <p class="is-size-6">
-          <span class="has-text-weight-bold">News:</span> 5.1.1 released. (January 17<sup>th</sup> 2020)
+          <span class="has-text-weight-bold">News:</span> 5.2.0 released. (March 5<sup>th</sup> 2020)
```
Comment (SmallJoker) [inline index.html:27]: April.

File: credits.html [hunk 2]
```diff
@@ -59,42 +60,53 @@ <h2>Contributors</h2>
...
+          <li>DS [Formspecs]</li>
+          <li>pyrollo [Formspecs: Hypertext]</li>
+          <li>v-rob [Formspecs]</li>
+          <li>Jordach [set_sky]</li>
+          <li>random-geek [Formspecs]</li>
+          <li>Wuzzy [Pathfinder, builtin]</li>
+          <li>ANAND (ClobberXD) [Fixes, per-player FOV]</li>
+          <li>Warr1024 [Fixes]</li>
+          <li>Paul Ouellette (pauloue) [Fixes, Script API]</li>
+          <li>Jean-Patrick G (kilbith) <jeanpatrick.guerrero@gmail.com> [Audiovisuals]</li>
```
Comment (sfan5) [inline credits.html:74]: missing escaping

Comment (rubenwardy) [reply credits.html:74]: oh, I didn't mean to add this

**Comment by rubenwardy** [comment]
I haven't managed to test this yet, because bundler/gem takes so fricking long! It's so much worse than npm

**Comment by Calinou** [comment]
@rubenwardy That's strange, it only takes about 1-2 minutes to install packages initially here.

**Comment by rubenwardy** [comment]
It takes 3m8s on my computer, and then doesn't even work

```
LoadError: Could not open library '/home/ruben/dev/web/minetest.github.io/vendor/bundle/ruby/2.7.0/gems/sassc-2.2.0/lib/sassc/libsass.so': /home/ruben/dev/web/minetest.github.io/vendor/bundle/ruby/2.7.0/gems/sassc-2.2.0/lib/sassc/libsass.so: cannot open shared object file: No such file or directory
  /home/ruben/dev/web/minetest.github.io/vendor/bundle/ruby/2.7.0/gems/ffi-1.11.1/lib/ffi/library.rb:145:in `block in ffi_lib'
  /home/ruben/dev/web/minetest.github.io/vendor/bundle/ruby/2.7.0/gems/ffi-1.11.1/lib/ffi/library.rb:99:in `map'
  /home/ruben/dev/web/minetest.github.io/vendor/bundle/ruby/2.7.0/gems/ffi-1.11.1/lib/ffi/library.rb:99:in `ffi_lib'
  /home/ruben/dev/web/minetest.github.io/vendor/bundle/ruby/2.7.0/gems/sassc-2.2.0/lib/sassc/native.rb:10:in `<module:Native>'
  /home/ruben/dev/web/minetest.github.io/vendor/bundle/ruby/2.7.0/gems/sassc-2.2.0/lib/sassc/native.rb:6:in `<module:SassC>'
  /home/ruben/dev/web/minetest.github.io/vendor/bundle/ruby/2.7.0/gems/sassc-2.2.0/lib/sassc/native.rb:5:in `<top (required)>'
  /home/ruben/dev/web/minetest.github.io/vendor/bundle/ruby/2.7.0/gems/sassc-2.2.0/lib/sassc.rb:31:in `require_relative'
```

I use  `bundle install --path vendor/bundle`, and have jekyll installed system wide (I'll try uninstalling it)

**Comment by rubenwardy** [comment]
Nope, same error

**Pull request merged**